bottles/backend/repos/repo.py

```diff
@@ -39,14 +40,12 @@ def __init__(self, url: str, index: str, offline: bool = False, callback=None):
         @GtkUtils.run_in_main_loop
         def set_catalog(result, error=None):
             self.catalog = result
-            RepoStatus.repo_done_operation(self.name + ".fetching")
+            EventManager.done(Events(self.name + ".fetching"))
```

Comment:
it's easy to make mistake in the future because of overlooking during making changes.
is there any alternative way to do this in a more explicit way, or can we at least add some guard code somewhere in `EventManager` to make sure we are crafting Enum object with correct names?

Reply:
If we try to craft an event that doesn't have a valid value, there will be a very clear error, so I think it's okay to let it like this:
```
>>> from enum import Enum
>>> class TestEnum(Enum):
...   Test1 = "test1"
...   Test2 = "test2"
... 
>>> TestEnum("test3")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: 'test3' is not a valid TestEnum
```

I'm not aware of better ways to do it as this code is very generic for good reasons, so imo it's okay, what do you think?

Reply:
looks good to me if it's already throwing error with invalid names